### PR TITLE
Better //c support (docs)

### PR DESCRIPTION
**Pull request by polluks**

### CODE REVIEW

File: doc/apple2enh.sgml [hunk 1]
```diff
@@ -472,6 +472,7 @@ BASIC.SYSTEM) there are some limitations for DOS 3.3:
 
 The enhanced&nbsp;Apple&nbsp;//e has no color text mode. Therefore the functions
 <tt/textcolor()/, <tt/bgcolor()/ and <tt/bordercolor()/ have no effect.
+The driver checks the Apple&nbsp;//c 80/40 switch on program startup.
```
Comment (oliverschmidt) [inline doc/apple2enh.sgml:475]: I think the term 'driver' doesn't apply well to CONIO.

**Comment by polluks** [comment]
Indeed, how about "constructor"?

File: doc/apple2enh.sgml (hunk 1 above)
Comment (greg-king5) [reply doc/apple2enh.sgml:475]: I suggest
```
<tt/cputc()/ checks the Apple&nbsp;//c 80/40 switch on program startup.
```

**Comment by oliverschmidt** [comment]
Documentation for https://github.com/cc65/cc65/pull/1026 which was just closed.

**Pull request closed without merging**